test/lint/lint-shell.sh

```diff
@@ -30,6 +30,7 @@ disabled=(
     SC2006 # Use $(..) instead of legacy `..`.
     SC2016 # Expressions don't expand in single quotes, use double quotes for that.
     SC2028 # echo won't expand escape sequences. Consider printf.
+    SC2039 # In POSIX sh, 'local' is undefined.
```

Comment:
I'm disabling this lint because in practice most POSIX-compatible shells support `local` (https://github.com/koalaman/shellcheck/issues/502#issuecomment-147463308).

Reply:
I think this lint covers more than `local`: https://github.com/koalaman/shellcheck/issues/1341 So it's not ideal to have to disable it. Especially since I think at least one of our uses of it is actually nonportable: it is nonportable when there is more than one local variable assigned, which appears to happen here:
```
./qa/zcash/checksec.sh:157:    local i ver1="${1}" ver2="${2}"
```
In any case, it looks as though these are the only scripts in which we use `local`:
```
qa/zcash/checksec.sh
test/lint/lint-locale-dependence.sh
zcutil/fetch-params.sh
```
and those could be fixed relatively easily.